File: app/src/main/java/com/github/brianspace/moviebrowser/di/modules/SettingsFragmentModule.java
package com.github.brianspace.moviebrowser.di.modules;

import com.github.brianspace.moviebrowser.ui.fragment.SettingsFragment;
import dagger.Module;
import dagger.android.ContributesAndroidInjector;

/**
 * Dagger module for SettingsFragment.
 */
@Module
public abstract class SettingsFragmentModule {
    @ContributesAndroidInjector
    /* default */ abstract SettingsFragment contributeSettingsFragmentInjector();
}

File: app/src/main/java/com/github/brianspace/moviebrowser/di/modules/SettingsFragmentModule.kt
package com.github.brianspace.moviebrowser.di.modules

import com.github.brianspace.moviebrowser.ui.fragment.SettingsFragment
import dagger.Module
import dagger.android.ContributesAndroidInjector

/**
 * Dagger module for SettingsFragment.
 */
@Module
abstract class SettingsFragmentModule {
    @ContributesAndroidInjector
    internal abstract fun contributeSettingsFragmentInjector(): SettingsFragment
}
